docs/source/testing/trip_test.rst

```diff
@@ -0,0 +1,99 @@
+
+The bucket test is the only one that fails in this instance. This is because the bucket model formulation was changed between these two commits, and so it is expected that the results with the two commits will differ. However, we note that the other tests have their results unchanged, meaning this is a safe modification to the code. Any unexpected failures should be investigated before submitting a pull-request.
+
+FAQs
+----
+
+* **What happens if the trip tests fail for my modifications?** If some of the trip tests fail, please investigate why you think this might be. If the tests fail on an isolated test case that you haven't used before, please post the results in your PR and we can discuss what is best to do. We will accept some PRs that fail the trip tests, but we will need to understand why first.
+
+* **When I submit a pull request, which two commit IDs should I use?** Please use the commit ID of the current HEAD of Isca's master branch as one, and the HEAD of your pull-requested branch as the other. *Please note that in order for this to work, you will have to use the* `-r` *option to select your own fork, as described above.*
+
+* **How long do the trip tests take to run?** The code is re-compiled several times during the trip tests, so the majority of time will be spent compiling. Each test case is only run for a few days, so once the code has compiled it shouldn't take more than 30 mins or so, depending on how many cores you're using. You can increase the number of cores by using the ``-n`` option.
+
+* **I am getting an error message saying that git cannot find my commit ID - how do I fix this?** If you are specifying commit IDs that are present on your fork but not in the ExeClim version of Isca, you will need to specify your own fork as the repository. To do this, use the ``-r`` option, e.g. ``-r git@github.com:sit23/Isca``. Note that you can specify the repository location using the HTTPS option, or the SSH option depending on your preference.
+
+* **Do I have to connect to GitHub to run the trip tests, or can I do it all locally?** It is possible to run the trip tests locally. Use the option ``-r PATH_TO_MY_LOCAL_REPO`` and it should work fine. 
+
+* **Which of the test cases does the trip tests run**? A list of the available test cases can be found in the ``list_all_test_cases_implemented_in_trip_test`` function in ``trip_test_functions.py``.
+
+* **I haven't setup Socrates - does that matter?** No - just post the results you can obtain.
+
+* **If I add a new test case to the** ``test_cases`` **folder, how do I add it to the trip tests?** In the ``get_nml_diag`` function in ``trip_test_functions.py``, you will see several similar calls to import the namelist dictionary from each of the existing test cases. Just copy this syntax and edit it for your test case. You can then add the name of your test case to the list in the ``list_all_test_cases_implemented_in_trip_test`` function, and it should be run by default.
```

Comment:
mention that their test will fail the trip test (as there is nothing to compare it to)

Reply:
Thanks - I've added this.